### PR TITLE
Setting up MSFS project XML to build package

### PR DESCRIPTION
**Pull request by ollyau**
- Add MSFS project structure
- Adds preliminary InGamePanel
- Adds AssetGroup for WASM module

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,7 @@
 *.user
 
 # Output
-Built/
\ No newline at end of file
+Built
```
Comment (StevenFrost) [inline .gitignore]: Accidentally removed the /?

Comment (ollyau) [reply .gitignore]: Oops!  Fixed! :)

File: Packages/PackageSources/meridian-simcache/html_ui/InGamePanels/SimCache/SimCache.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <script type="text/javascript" src="/JS/coherent.js"></script>
+    <script type="text/javascript" src="/JS/common.js"></script>
+    <script type="text/javascript" src="/JS/dataStorage.js"></script>
+    <script type="text/javascript" src="/JS/buttons.js"></script>
+    <script type="text/javascript" src="/JS/Services/ToolBarPanels.js"></script>
+    <script type="text/javascript" src="/JS/SimVar.js"></script>
+
+    <link rel="import" href="/templates/NewPushButton/NewPushButton.html" />
+    <!-- <link rel="import" href="/templates/ToggleButton/toggleButton.html" /> -->
+    <!-- <link rel="import" href="/templates/tabMenu/tabMenu.html" /> -->
```
Comment (StevenFrost) [inline Packages/PackageSources/meridian-simcache/html_ui/InGamePanels/SimCache/SimCache.html]: Can we remove this commented out code?

Comment (ollyau) [reply Packages/PackageSources/meridian-simcache/html_ui/InGamePanels/SimCache/SimCache.html]: Sure!

**Comment by StevenFrost** [comment]
Looks good to me!

**Review by StevenFrost — APPROVED** [review]

**Pull request merged**